src/main/kotlin/no/nav/klage/kaka/api/view/KodeverkResponse.kt

```diff
@@ -18,40 +23,76 @@ fun getYtelser(): List<YtelseKode> =
             id = ytelse.id,
             navn = ytelse.navn,
             beskrivelse = ytelse.beskrivelse,
-            hjemler = hjemlerPerYtelse[ytelse]?.map { it.toDto() } ?: emptyList(),
+            hjemler = ytelseToHjemler[ytelse] ?: emptyList(),
+            lovKildeToRegistreringshjemler = ytelseToLovKildeToRegistreringshjemmel[ytelse] ?: emptyList(),
             enheter = enheterPerYtelse[ytelse]?.map { it.toDto() } ?: emptyList(),
         )
     }
 
-val enheterPerYtelse: Map<Ytelse, List<Kode>> = Ytelse.values().associateWith {
-    listOf(Enhet.NAV_MOSS, Enhet.NAV_XXXX, Enhet.NAV_YYYY)
+fun getRegistreringshjemler(): List<KodeDto> =
+        Registreringshjemmel.values().map {
+            KodeDto(
+                id = it.id,
+                navn = it.lovKilde.beskrivelse + " - " + it.spesifikasjon,
+                beskrivelse = it.lovKilde.navn + " - " + it.spesifikasjon,
+            )
+        }
+
+val ytelseToLovKildeToRegistreringshjemmel: Map<Ytelse, List<LovKildeToRegistreringshjemler>> = mapOf(
```

Comment:
Er dette noe som skulle vært i kodeverket?

Reply:
Det er noe logikk inne der som er ment for å tilpasse dataene til det frontend trenger, med disse DTOene. Det kan nok tenkes at hele kan flyttes til kodeverket etter hvert, men her var det litt pågående arbeid sammen med frontend for å komme frem til en form som de kan bruke. Tenker det bør flyttes over til kodeverk etter hvert, men er fornøyd med å ha det her foreløbig.